src/view/trip-info-view.js

```diff
@@ -0,0 +1,67 @@
+import AbstractView from '../framework/view/abstract-view.js';
+import { humanizePointDate, sortPointsByDate } from '../utils.js';
+
+const createTripInfoTitle = (points) => {
+  const startTrip = points[0].destination.name;
+  const finishTrip = points[points.length-1].destination.name;
```

Comment:
```suggestion
  const finishTrip = points[points.length - 1].destination.name;
```
Так же чтобы получить последний элемент можно воспользоваться новым синтаксисом .at()